test/e2e/specs/sidebar-behaviour.test.js

```diff
@@ -66,8 +61,7 @@ describe( 'Publishing', () => {
 
 	it( 'Should reopen sidebar the sidebar when resizing from mobile to desktop if the sidebar was closed automatically', async () => {
 		await setViewport( 'large' );
-		await newPost();
-		await setViewport( 'small' );
+		await newPost( { viewport: 'small' } );
```

Comment:
I think it's weird to have an option to set the viewport size as part of the `newPost` helper, I prefer a separate helper for that.

Reply:
I can revert those changes. Previously, we were setting `large` viewport using `newDesktopBrowserPage` helper.

Reply:
Reverted 👍